react-components/src/analysis/view/dialogs/ShareWithSupportDialog.js

```diff
@@ -188,73 +181,62 @@ class ShareWithSupportDialog extends React.Component {
                             }
                         }/>
                     <DialogContent>
-                        <div style={{
-                            display: shareWithSupport ?
-                                "none" :
-                                "block"
-                        }}>
-                            <div style={{
-                                display: status === analysisStatus.COMPLETED ?
-                                    "block" :
-                                    "none"
-                            }}>
+                        {!shareWithSupport &&
+                        <React.Fragment>
+
+                            {status === analysisStatus.COMPLETED &&
+                            <React.Fragment>
```

Comment:
These `React.Fragment` tags inside these `status === analysisStatus.XXX && ` blocks aren't required, since they're only wrapping 1 component (only `CompletedStateCondition` is inside this one, for example).